tests/xtro-sharpie/iOS-UIKit.todo

```diff
@@ -1,3 +0,0 @@
-!missing-protocol-conformance! NSShadow should conform to NSSecureCoding
```

Comment:
if **Intro** did not spot that it's because it's been added (in beta 2) as an exception
please updated `tests/introspection/ApiProtocolTest.cs` to remove it

Reply:
Looking at the diff, it was added in beta 3 => https://github.com/xamarin/xamarin-macios/wiki/UIKit-iOS-xcode10-beta3

Reply:
@mandel-macaque yes, the header was updated in beta 3, but that was not my point :)

The fact that **intro** (not **xtro**) did not spot it means it was excluded earlier - because it was conforming but **not** in the headers. Now that the headers are updated then that special case, in **Intro**, needs to be removed.

Reply:
Oh, got you.